website/content/project/chiventure.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "chiventure: a Text Adventure Game Engine"
+type: page
+---
+
+```
+You wake up to find yourself in a strange hall. You have no idea how you got here. The last thing you remember is a blinding flash of light. You look at the tapestries lining the walls for any clues on your whereabouts. The tapestries depict valiant knights, fire-breathing wyverns, armies of goblins, and fearsome rulers sitting in their gilded thrones. Wherever you are, it must be in a different world from your own. 
+
+No, not just a different world, but a different time altogether. 
+
+A time of wizards and warlords and all manner of magical creatures, a time to fight back the forces of Throxar the Terrible and to heed the counsel of Wilfrax the Wise... 
+
+It is a time of adventure. 
+
+It is the time of...
+```
+
+[![chiventure](/cmsc22000/img/chiventure.jpg "chiventure")](https://www.youtube.com/watch?v=DGSXURPvRxY)
+
+
+The CMSC 22000 project revolves around implementing a _text adventure game engine_. This means that you won’t be implementing an individual game but, rather, a platform that allows game authors (with little or no knowledge of computer programming) to develop text adventure games.
+
+Not just that, you won't be implementing it from scratch but, rather, will work on existing code that was written in previous editions of CMSC 22000.
```

Comment:
*previous iterations (?)

Reply:
*previous years (?)

Reply:
I changed it to "previous offerings" (that seems to work better when referring to a course)